src/Products/Annotation/Annotator/WatermarkAnnotator.cs

```diff
@@ -1,57 +1,62 @@
-﻿using GroupDocs.Annotation.Domain;
-using GroupDocs.Annotation.MVC.Products.Annotation.Entity.Web;
+﻿using GroupDocs.Annotation.MVC.Products.Annotation.Entity.Web;
 using System;
+using GroupDocs.Annotation.Options;
+using GroupDocs.Annotation.Models.AnnotationModels;
+using GroupDocs.Annotation.Models;
 
 namespace GroupDocs.Annotation.MVC.Products.Annotation.Annotator
 {
-    public class WatermarkAnnotator : AbstractTextAnnotator
+    public class WatermarkAnnotator : BaseAnnotator
     {
+        private WatermarkAnnotation watermarkAnnotation;
 
-        public WatermarkAnnotator(AnnotationDataEntity annotationData, PageData pageData)
-            : base(annotationData, pageData)
+        public WatermarkAnnotator(AnnotationDataEntity annotationData, PageInfo pageInfo)
+            : base(annotationData, pageInfo)
         {
+            watermarkAnnotation = new WatermarkAnnotation()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove these redundant parentheses.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372470)